modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -13,8 +18,14 @@ def submit
             raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Claim not found')
           end
 
+          unless bgs_claim.dig(:bnft_claim_dto, :filed5103_waiver_ind) == FILE_5103
```

Comment:
no changes needed currently, as I'm not sure the best approach, but I just noticed that one of the claims I searched for did not have a `filed5103_waiver_ind` field on it. If you hit
`bgs_claim.dig(:bnft_claim_dto, :filed5103_waiver_ind)` it would return `nil`. That being said, it still accepts the update and the change to BGS to move a non-existent value to "Y". 

I'm curious here, and maybe tagging in @acovrig on this would make sense. What would happen if we didn't have a value on a bgs-ext `update` call? For example, say we're missing a different field (maybe `bnft_claim_type_label`) that we aren't appending to the object in this particular flow - would bgs send us an error saying it's missing a field? would it corrupt data? Maybe I'm over thinking this currently but I want to handle all basis here. 

If we were to receive an error from BGS, I'm assuming our code would just send their error message back?